agents/backport_agent.py

```diff
@@ -513,6 +513,8 @@ class State(PackageUpdateState):
         attempts_remaining: int = Field(default=max_build_attempts)
         used_cherry_pick_workflow: bool = Field(default=False)  # Track if cherry-pick was used
         incremental_fix_attempts: int = Field(default=0)  # Track how many times we tried incremental fix
+        merge_request_url: str | None = Field(default=None)
```

Comment:
I think this is already present in `PackageUpdateState`, and `merge_request_newly_created` should probably be there as well.